src/test/java/de/retest/recheck/cli/subcommands/CommitIT.java

```diff
@@ -94,9 +96,9 @@ public void commit_should_handle_sut_state_files_that_cannot_be_found() throws E
 
 		cut.run();
 
-		final String expectedMessage = "The Golden Master 'goldenMaster' cannot be found.\n" //
+		final String expectedMessage = "The Golden Master 'goldenMaster' cannot be found." + lineSeparator() //
```

Comment:
Always these os specific stuff ... Do we have access/want to copy `ApprovalsUtils`?